frontend/components/how-it-work/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { GradientText, Heading, HEADING_OPTIONS } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const HowItWork = ({ title, paragraph, cards, sideImage, steps }) => {
```

Comment:
small update would be nice just add data and destructuree it inside the component instead of taking so many props.